src/controller/tests/TestCommissioningWindowOpener.cpp

```diff
@@ -188,5 +188,23 @@ TEST_F(TestCommissioningWindowOpener, OpenCommissioningWindowPasscode_Failure_In
     EXPECT_EQ(err, CHIP_ERROR_INVALID_ARGUMENT);
 }
 
+#if CHIP_DEVICE_CONFIG_ENABLE_JOINT_FABRIC
+TEST_F(TestCommissioningWindowOpener, OpenJointCommissioningWindowPasscode_Success_NoPin)
+{
+    SetupPayload ignored;
+    Callback::Callback<Controller::OnOpenCommissioningWindow> callback(OCWPasscodeCallback, this);
+    CHIP_ERROR err = opener.OpenCommissioningWindow(Controller::CommissioningWindowPasscodeParams()
```

Comment:
Test declaration (4 lines above) states that this is a unit test for OJCW for the case when no pin is provided. However, line above seems to test regular OCW when no pin provided.

Reply:
How is the test case supposed to validate OJCW if `opener.OpenCommissioningWindow`? Shouldn't it be instead `opener.OpenJointCommissioningWindow`?

Reply:
The test validates OJCW by calling opener.OpenCommissioningWindow with the last parameter set to true. This parameter determines whether a regular OCW or an OJCW command is sent.

Reply:
Thanks for this explanation, I was misreading the and I somehow I missed the additional parameter for the existing OpenCommissioningWindow. Resolving the conversation.